locale/en/docs/guides/security/index.md

```diff
@@ -0,0 +1,427 @@
+not on the request's contents, but on the timing and pattern of the requests
+being sent to the server.
+
+**Mitigations**
+
+* Use a reverse proxy to receive and forward requests to the Node.js application.
+Reverse proxies can provide caching, load balancing, IP blacklisting, etc. which
+reduce the probability of a DoS attack being effective.
+* Correctly configure the server timeouts, so that connections that are idle or
+where requests are arriving too slowly can be dropped. See the different timeouts
+in [`http.Server`][], particularly `headersTimeout`, `requestTimeout`, `timeout`,
+and `keepAliveTimeout`.
+* Limit the number of open sockets per host and in total. See the [http docs][],
+particularly `agent.maxSockets`, `agent.maxTotalSockets`, `agent.maxFreeSockets`
+and `server.maxRequestsPerSocket`.
+
+### DNS Rebinding (CWE-346)
+
+This is an attack that can target Node.js applications being run with the
+debugging inspector enabled using the [--inspect switch][].
+
+Since websites opened in a web browser can make WebSocket and HTTP requests,
+they can target the debugging inspector running locally.
+This is usually prevented by the [same-origin policy][] implemented by modern
+browsers, which forbids scripts from reaching resources from different origins
+(meaning a malicious website cannot read data requested from a local IP address).
+
+However, through DNS rebinding, an attacker can temporarily control the origin
+for their requests so that they seem to originate from a local IP address.
+This is done by controlling both a website and the DNS server used to resolve
+its IP address, see [DNS Rebinding wiki][] for more details.
+
+This is not considered a vulnerability in Node.js itself, unless the request has
+an invalid origin (such as an invalid IP, or any domain other than localhost)
+and Node tries to resolve it via the malicious DNS server.
+The expected behavior is that those requests are ignored instead of processed.
+
+**Mitigations**
+
+* Disable inspector on _SIGUSR1_ signal by attaching a `process.on(‘SIGUSR1’, …)`
+listener to it.
+* Do not run the inspector protocol in production.
+
+### Exposure of Sensitive Information to an Unauthorized Actor (CWE-552)
+
+All the files and folders included in the current directory are pushed to the
+npm registry during the package publication.
+
+There are some mechanism to control this behavior by defining a blocklist with
+`.npmignore` and `.gitignore` or by defining an allowlist in the `package.json`
+
+**Mitigation**
+
+Using `npm publish –dry-run` list all the files to publish. Ensure to review the
+content before publishing the package.
+
+It’s also important to create and maintain ignore files such as `.gitignore` and
+`.npmignore`.
+Throughout these files, you can specify which files/folders should not be published.
+The [files property][] in `package.json` allows the inverse operation
+-- allowed list.
+In case of an exposure, make sure to [unpublish the package][].
```

Comment:
Style suggestion to follow the convention for bullet points in Mitigation lists
```suggestion
* Using `npm publish –dry-run` list all the files to publish. Ensure to review the
content before publishing the package.
* It’s also important to create and maintain ignore files such as `.gitignore` and
`.npmignore`.
Throughout these files, you can specify which files/folders should not be published.
The [files property][] in `package.json` allows the inverse operation
-- allowed list.
* In case of an exposure, make sure to [unpublish the package][].
```